docs/content/en/docs/administration/permissions/list_dashboard_permissions.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "list_dashboard_permissions"
+linkTitle: "list_dashboard_permissions"
+weight: 16
+no_list: true
+superheading: "catalog_permission."
+---
+
+
+
+``list_dashboard_permissions(workspace_id: str, dashboard_id: str) -> CatalogDashboardPermissions``
+
+Provide managing dashboard permissions for user and user groups.
```

Comment:
This one lists the permissions of users and user groups that have access to the dashboard in the workspace, and any permission rules in effect for the dashboard and workspace.